pom.xml

```diff
@@ -61,7 +61,7 @@
 
         <!-- Dependency version properties -->
         <version.httpclient>4.5.13</version.httpclient>
-        <version.openshift-client>5.12.2</version.openshift-client>
+        <version.openshift-client>6.4.1</version.openshift-client>
```

Comment:
Latest version is `6.7.2`. Would it be possible to update to this version?

Reply:
Should be possible, but there were some changes to the client again. I'll see if I can get it to work with them.